### PR TITLE
Global Alert - show the actual error message in the banner (#1019 #1028)

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by marcinbodnar** [comment]
@kcole16 @ilblackdragon 
Wallet already has a functionality to show proper error in Global Alert banner - but the error from RPC need's to have a `type` property. For example, error, when there is a lack of storage balance to cover adding another 2FA confirmation, seems not to have `type`, so it's showing default error. 

With this change, all Global Alerts will also show the actual error message.

Ideally, all RPC errors should have a `type` property, and in Wallet, we should have all these types listed in translations files.

**Comment by marcinbodnar** [comment]
@kcole16 @corwinharrell 
A new Global Alert is ready. I prepare it from scratch, so it's not using semantic ui anymore.

**Comment by corwinharrell** [comment]
> it's not using semantic ui anymore

Praise god

**Comment by corwinharrell** [comment]
@marcinbodnar this looks fantastic. Great job. 🚀

**Review by corwinharrell — APPROVED** [review]
@marcinbodnar this looks fantastic. Great job. 🚀

**Comment by Patrick1904** [comment]
Love this! Big improvement. Just a couple of thoughts:
1. After being redirected back to profile post recovery setup, the global alert only shows for ~0.5 
sec before disappearing (might be an existing issue). Can we fix that too?
2. Can we vertically align the global alert icon, it's just slightly off (please see attached)
3. Is there a reason for introducing a new font? (cc @corwinharrell). I am working on replacing our current font with one global font (Inter) [here](https://github.com/near/near-wallet/pull/1011)

Screen Shot 2020-10-16 at 11 39 47 AM

**Comment by corwinharrell** [comment]
@Patrick1904 The only new font is the one being used for the raw RPC output that I'd like to be monospace. Are we using a monospace typeface elsewhere? If so we can default to what we're already using.

**Comment by marcinbodnar** [comment]
@Patrick1904 
> After being redirected back to profile post recovery setup, the global alert only shows for ~0.5
sec before disappearing (might be an existing issue). Can we fix that too?

What kind of recovery did you use?

> Can we vertically align the global alert icon, it's just slightly off (please see attached)

In this situation, when there is no text it looks off, but it looks good that way if there is a text. I wonder, what is the situation when only 'Success` header is displayed?

File: src/components/responsive/GlobalAlert.js [hunk 1]
```diff
@@ -1,150 +1,177 @@
...
-                padding: 0px;
-                cursor: pointer;
-
-                &.white {
-                    filter: brightness(0) invert(1);
-                }
-            }
+const Content = styled.div`
+    min-height: 74px;
+    max-width: 500px;
```
Comment (corwinharrell) [inline src/components/responsive/GlobalAlert.js:58]: Let's go ahead and add `min-width: 275px;`

**Review by corwinharrell — CHANGES_REQUESTED** [review]
Just thought it would be nice for alerts lacking content to not have to be so small, so suggested the addition of a `min-width`.

**Comment by Patrick1904** [comment]
> @Patrick1904
> 
> > After being redirected back to profile post recovery setup, the global alert only shows for ~0.5
> > sec before disappearing (might be an existing issue). Can we fix that too?
> 
> What kind of recovery did you use?

Whenever I am logged in and start the recovery setup flow from the profile, and then get redirected back to the profile


> > Can we vertically align the global alert icon, it's just slightly off (please see attached)
> 
> In this situation, when there is no text it looks off, but it looks good that way if there is a text. I wonder, what is the situation when only 'Success` header is displayed?

I believe this was when I disabled seed phrase from the profile. Might want to align the icon vertically with the main heading/title (cc @corwinharrell)

**Comment by marcinbodnar** [comment]
@Patrick1904 @corwinharrell 
Bug with disappearing alert after setting up recovery was not related to these changes. I fixed it for seed phrase, phone, and email.

Also adjusted Alert with only header visible.

**Review by corwinharrell — APPROVED** [review]

**Review by Patrick1904 — APPROVED** [review]

**Pull request merged**